test/functional/commands/jetpack-compose-e2e-specs.js

```diff
@@ -10,14 +10,15 @@ describe('Jetpack Compose', function () {
   this.timeout(MOCHA_TIMEOUT);
 
   let driver;
-  before(async function () {
+  beforeEach(async function () {
     // For SDK 23 and below Jetpack compose app crashes while running under instrumentation.
     if (parseInt(process.env.ANDROID_SDK_VERSION, 10) <= 23) {
       return this.skip();
     }
     driver = await initSession(COMPOSE_CAPS);
```

Comment:
this might add to the suite duration, but I'm fine with that, since we don't have many tests in this suite